src/app/analyze/analyze-overview/analyze-overview.component.html

```diff
@@ -146,6 +146,215 @@ <h2 class="panel-title text-center">Hypotheses Evaluating</h2>
             </div>
         </div>
     </div>
+</div>-->
+<div id="analyze-overview" class="container-fluid analyze-overview-wrapper">
+  <section class="row row-cards-pf space-overview-wrapper">
+    <div class="flex-grow-4">
+      <aside>
+        <div class="card-pf">
+          <div class="flex-grow-1">
+            <h2 class="card-pf-title">BalloonPopGame</h2>
+            <p class="margin-bottom-0">Description of the "SPACE" goes here.</p>
```

Comment:
why is the default `<p>` being overwritten? The visuals just state this as a `<p>` with no alterations.